src/Components/Components/src/Rendering/HtmlRenderer.cs

```diff
@@ -224,7 +223,17 @@ private int RenderChildren(List<string> result, ArrayRange<RenderTreeFrame> fram
             return position + maxElements;
         }
 
-        private async Task<(int, ArrayRange<RenderTreeFrame>)> CreateInitialRenderAsync(Type componentType, ParameterCollection initialParameters)
+        private async Task<(int, ArrayRange<RenderTreeFrame>)> CreateInitialRenderAsync(Type componentType, ParameterView initialParameters)
+        {
+            var component = InstantiateComponent(componentType);
+            var componentId = AssignRootComponentId(component);
+
+            await RenderRootComponentAsync(componentId, initialParameters);
+
+            return (componentId, GetCurrentRenderTreeFrames(componentId));
+        }
+
+        private async Task<(int, ArrayRange<RenderTreeFrame>)> CreateInitialRenderCoreAsync(Type componentType, ParameterView initialParameters)
```

Comment:
Is this a hangover from the investigation about ref structs? This method doesn't seem to be used.

Reply:
ah yes, this can be removed. I think this should be the only once since my investigation of ref struct was short.